app/assets/stylesheets/common/modal/modal-overrides.scss

```diff
@@ -265,8 +265,8 @@
 }
 
 .d-modal.chat-modal-new-message {
+  align-items: flex-start;
   .d-modal {
```

Comment:
Isn't it odd to have a `.d-modal` as a child of `.d-modal` in the stylesheet?

Reply:
No, this compiles to:

```
.d-modal.chat-modal-new-message {
  .d-modal__container {}
etc
}
```

.d-modal itself isnt declared more than once

Reply:
Oh I see... ok. I think my personal preference would be to use just that 

```
.d-modal.chat-modal-new-message {
  .d-modal__container {}
}
```

vs 

```
.d-modal.chat-modal-new-message {
  .d-modal {
    &__container {}
  }
}
```

But not a major issue.